### PR TITLE
Fix Pylint warnings in unit tests

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: tests/unit_tests/test_kronos_gnosis_package_topic_model.py [hunk 1]
```diff
@@ -21,15 +21,18 @@ def test_generate_and_save_package_topic_model_local(self):
             "tests/data/data_gnosis/output-ptm-data")
         self.assertTrue(output_data_store is not None)
 
-        package_topic_model = GnosisPackageTopicModel.curate(data_store=input_data_store,
-                                                             filename="data_input_curated_package_topic/package_topic.json")
+        package_topic_model = GnosisPackageTopicModel.curate(
+            data_store=input_data_store,
+            ilename="data_input_curated_package_topic/package_topic.json")
```
Comment (fridex) [inline tests/unit_tests/test_kronos_gnosis_package_topic_model.py]:
```python
filename="data_input_curated_package_topic/package_topic.json")
```
(**f** was removed)

File: tests/unit_tests/test_kronos_pgm_pomegranate.py [hunk 2]
```diff
@@ -42,8 +42,8 @@ def test_train_and_save_kronos_list_local(self):
                 kronos_model = PGMPomegranate.train(kronos_dependency_dict=kronos_dependency_dict,
                                                     package_occurrence_df=cooccurrence_matrix_df)
                 self.assertTrue(kronos_model is not None)
-                filename = "data_kronos_user_eco" + "/" + str(user_category) + "/" + "kronos" + "_" + str(
-                    ecosystem) + ".json"
+                filename = "data_kronos_user_eco" + "/" + str(user_category) + "/" + "kronos" + \
+                    "_" + str(ecosystem) + ".json"
```
Comment (fridex) [inline tests/unit_tests/test_kronos_pgm_pomegranate.py]:
```python
import os
os.path.join("data_kronos_user_eco", str(user_category), "kronos", "_" + str(ecosystem) + ".json")
```

Comment (tisnik) [reply tests/unit_tests/test_kronos_pgm_pomegranate.py]: nice catch

**Comment by tisnik** [comment]
[test]

**Pull request merged**